core/src/models/Export.ts

```diff
@@ -304,16 +304,7 @@ export class Export extends CommonModel<Export> {
       ["canceled", "failed", "complete"].includes(instance.state)
     ) {
       const exportData = {
-        id: instance.id,
-        state: instance.state,
-        recordId: instance.recordId,
-        modelId: (await instance.$get("record")).modelId,
-        destinationId: instance.destinationId,
-        newRecordProperties: instance.newRecordProperties,
-        newGroups: instance.newGroups,
-        toDelete: instance.toDelete,
-        errorMessage: instance.errorMessage,
-        errorLevel: instance.errorLevel,
+        ...(await instance.apiData(false)),
         timestamp: new Date().toISOString(),
```

Comment:
Thoughts about this additional timestamp property? Wanted to keep the file easy to parse (just read each line as JSON) but it means that unlike everything else this isn't a field on the Export.

Reply:
I do like that - so the line would be `{... export.apiData(), timestamp: new Date().toISOString()}`